app/controllers/samples_controller.rb

```diff
@@ -680,56 +680,65 @@ def save_metadata_v2
     end
   end
 
+  # GET /samples/1/legacy
+  def legacy
+    redirect_to action: "show", legacy: true
+  end
+
   # GET /samples/1
   # GET /samples/1.json
   def show
-    @pipeline_run = select_pipeline_run(@sample, params[:pipeline_version])
-    @amr_counts = nil
-    can_see_amr = (current_user.admin? || current_user.allowed_feature_list.include?("AMR"))
-    if can_see_amr && @pipeline_run
-      amr_state = @pipeline_run.output_states.find_by(output: "amr_counts")
-      if amr_state.present? && amr_state.state == PipelineRun::STATUS_LOADED
-        @amr_counts = @pipeline_run.amr_counts
+    if !params[:legacy] && (current_user.admin? || current_user.allowed_feature_list.include?("report_v2"))
```

Comment:
Let's just do feature flag and then we add the flag to admins. I just want to make sure we can revert this quickly without an extra push.

Reply:
Removed admin access and added link to legacy page.